project-reports/2024/2024-annual-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,100 @@
+
+Initially, Iroha 2 used Ursa as a core cryptographic library to leverage its secure and reusable features. In turn, Ursa worked as an abstraction over other cryptography libraries. Iroha 2 benefited from Ursa's high cryptographic security standards. This change also minimised the need for redundant cryptographic code.
+
+At this point, the necessary parts of Ursa are detached and fully integrated. The refactoring of the [`iroha-crypto`](https://github.com/hyperledger/iroha/tree/iroha2-dev/crypto) library itself continues.
+
+The latest topic in Iroha's cryptographic infrastructure addressed by our team is [#4181](https://github.com/hyperledger/iroha/pull/4181); it allows the parsed keys to be stored instead of the raw bytes.
+
+## Next Year's Goals
+
+Our primary goal is to build upon the achievements of 2024 by successfully delivering the Hyperledger Iroha 2.0 MVP and stabilising its SDKs.
+
+Beyond this milestone, our focus with the development team will address functional deliverables not included in the MVP but essential for the project's completeness in the post-production phase. 
+
+Furthermore, we aspire to begin developing of architectural upgrades, laying the groundwork for delivering Iroha 2.0 as a public blockchain. It is important to note that Iroha 2 currently operates as a private blockchain, and the shift to a public blockchain will require dedicated resources for research and development. Additionally, we plan to incorporate a set of additional functional features in the post-release phase, strategically supporting the commercial efforts of Central Bank Digital Currency Proof of Concepts. We aim to advance our project's technical capabilities and foster its broader adoption and utility within the industry.
+
+Our community has expressed an interest in using Iroha's codebase as something that can reliably run on the embedded SoCs, similar to those found in modern mobile phones. They could compile Iroha on different ARM SoCs with limited RAM and help test it in new conditions. While this isn't a priority, Rust allows such ports to function well, and the main concern would be overall performance. We are engaged in dialogue with those users to benchmark Iroha and expand its use scope.
+
+We are also excited about the opportunity to participate in the Hyperledger [AI FAQ](https://github.com/hyperledger-labs/aifaq) project and contribute our documentation.
+
+# Help Required
+
+Since we expect Iroha 2 to grow, we would like to request support for our PR effort, intending to report the project's results. For now, the projects we can mention are [Bokolo Cash](https://soramitsu.co.jp/bokolo-cash), Bakong, and [Fraud Intelligence Limited](https://soramitsu.co.jp/fil-2023-in-review).
```

Comment:
Nice to have a link for Bakong -- perhaps this?  https://bakong.nbc.gov.kh/download/NBC_BAKONG_White_Paper.pdf. Or is there something that talks more specifically about the use Iroha?